packages/overlays/README.md

```diff
@@ -23,18 +34,43 @@ npm i --save @lion/overlays
 ### Example
 
 ```js
-import { overlays } from '@lion/overlays';
-
-const myCtrl = overlays.add(
-  new OverlayTypeController({
-    /* options */
-  }),
-);
-// name OverlayTypeController is for illustration purpose only
-// please read below about existing classes for different types of overlays
+import '@lion/overlays/lion-overlay.js';
+
+html`
+  <lion-overlay>
+    <div slot="content" class="tooltip" .config=${{
```

Comment:
config on the div?

Reply:
Probably unintended mistake, config should be on the host element 🙈